From b87809d7ec2a88ec1f7a38e48c0f985cdeac5a21 Mon Sep 17 00:00:00 2001
From: nojick <nojick@gmail.com>
Date: Thu, 11 Jul 2024 14:46:51 +0300
Subject: [PATCH] glodroid:hal usb fixup aosp

glodroid/configuration/hals/usb-gadget/lib/MonitorFfs.cpp:62:16: error: unqualif
ied call to 'std::move' [-Werror,-Wunqualified-std-cast-call]
    mEpollFd = move(epollFd);
               ^
               std::
---
 hals/usb-gadget/lib/MonitorFfs.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hals/usb-gadget/lib/MonitorFfs.cpp b/hals/usb-gadget/lib/MonitorFfs.cpp
index 0cdf038..6d09a8a 100644
--- a/hals/usb-gadget/lib/MonitorFfs.cpp
+++ b/hals/usb-gadget/lib/MonitorFfs.cpp
@@ -59,9 +59,9 @@ MonitorFfs::MonitorFfs(const char* const gadget)
 
     if (addEpollFd(epollFd, eventFd) == -1) abort();
 
-    mEpollFd = move(epollFd);
-    mInotifyFd = move(inotifyFd);
-    mEventFd = move(eventFd);
+    mEpollFd = std::move(epollFd);
+    mInotifyFd = std::move(inotifyFd);
+    mEventFd = std::move(eventFd);
     gadgetPullup = false;
 }
 
-- 
2.39.2

